tests/Cms/Pages/PageTest.php

```diff
@@ -1111,4 +1111,69 @@ public function testPageModel()
 
         Page::$models = [];
     }
+
+    public function testPageController()
```

Comment:
The test method should be called like the tested method:

```suggestion
    public function testController()
```